scripts/globals/spells/trust/naji.lua

```diff
@@ -21,28 +25,11 @@ function onSpellCast(caster, target, spell)
 end
 
 function onMobSpawn(mob)
-    mob:addListener("ENMITY_CHANGED", "NAJI_ENMITY_CHANGED", function(trust, master, target)
-        if trust:getID() ~= target:getID() then
-            trust:setLocalVar("tryProvoke", 1)
-        end
-    end)
-
-    mob:addListener("COMBAT_TICK", "NAJI_COMBAT_TICK", function(trust, master, target)
-        local tryProvoke = trust:getLocalVar("tryProvoke")
-        local provokeReadyTime = trust:getLocalVar("provokeReadyTime")
-
-        if tryProvoke == 1 and os.time() >= provokeReadyTime then
-            trust:useJobAbility(19, target)
-            trust:setLocalVar("tryProvoke", 0)
-            trust:setLocalVar("provokeReadyTime", os.time() + 30)
-        end
-
-        if trust:getTP() >= 1000 then
-            local weaponSkills = {33, 34, 40}
-            local ws = weaponSkills[math.random(1, #weaponSkills)]
-            trust:useMobAbility(ws, target)
-        end
-    end)
+    mob:addSimpleGambit(ai.t.SELF, ai.c.NOT_HAS_ENMITY, 0,
```

Comment:
I worry that NOT_HAS_ENMITY might be confused for not having _any_ enmity. Uncertain about best descriptor. NOT_MOB_TARGET is the best I've got.

NOT_ENEMY_TARGET?

Reply:
I see it, now that you mention it